simplq/src/components/common/QR_Scanner/QR_Scanner.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react'
+import {Fab, TextareaAutosize} from '@material-ui/core'
+import {ArrowBack} from '@material-ui/icons'
```

Comment:
This QRScanner is entirely new and is not mentioned in the PR description. Can you give details on how it works and how it is integrated?

Reply:
Actually, as I finished with #601 and #636 , I started moving with #471 . After I implemented some code, I have to solve the changes that you requested and then I committed it.

I am extremely sorry for this issue. I have updated the PR description now. It basically solves #471 .